.eslintrc.js

```diff
@@ -78,12 +78,12 @@ module.exports = {
     'prefer-destructuring': 0,
 
     /* tsx */
-    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-explicit-any': 0,
     '@typescript-eslint/no-unused-vars': 'warn',
     '@typescript-eslint/no-var-requires': 'warn',
     '@typescript-eslint/no-empty-function': 0,
     '@typescript-eslint/ban-types': 'warn',
-    '@typescript-eslint/ban-ts-comment': 'warn',
+    '@typescript-eslint/ban-ts-comment': 0,
```

Comment:
this is not fixing the warnings, it's suppressing them. Please remove from this PR.

Reply:
I will going to work on this PR gradually, currently I'm working on cleaning the ban-types. Although, the ban-ts comment only used three time on project and it used only to suppress `react-vis` ts problem.